test/Microsoft.AspNetCore.WebSockets.ConformanceTest/Autobahn/AutobahnTester.cs

```diff
@@ -89,8 +85,7 @@ public void Verify(AutobahnResult result)
 
         public async Task DeployTestAndAddToSpec(ServerType server, bool ssl, string environment, CancellationToken cancellationToken, Action<AutobahnExpectations> expectationConfig = null)
         {
-            var port = Interlocked.Increment(ref _nextPort);
-            var baseUrl = ssl ? $"https://localhost:{port}" : $"http://localhost:{port}";
+            var baseUrl = ssl ? "https://localhost:0" : "http://localhost:0";
```

Comment:
Just to confirm, "localhost:0" means for a port to be selected by the system right?